code/game/machinery/teleporter.dm

```diff
@@ -302,24 +400,19 @@
 	return
 */
 
-/obj/machinery/teleport/station
-	name = "station"
-	desc = "It's the station thingy of a teleport thingy." //seriously, wtf.
-	icon_state = "controller"
-	var/active = 0
-	var/engaged = 0
-	use_power = 1
-	idle_power_usage = 10
-	active_power_usage = 2000
-	var/obj/machinery/teleport/hub/com
+
 
 /obj/machinery/teleport/station/New()
 	..()
 	overlays.Cut()
 	overlays += "controller-wires"
 
-/obj/machinery/teleport/station/attackby(var/obj/item/weapon/W)
-	src.attack_hand()
+/*
+/obj/machinery/teleport/station/default_deconstruction(obj/item/I, mob/user)
+	if(engaged)
+		src.disengage()
+	. = ..()*/
+	
```

Comment:
There's no need to comment them out

Reply:
Ah dang it, that's old part of code i forgot to clean up